app/controllers/users_controller.rb

```diff
@@ -21,9 +20,19 @@ def update
     redirect_to :back
   end
 
+  #def destroy
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.